core/src/main/java/org/openedx/core/domain/model/CourseDateBlock.kt

```diff
@@ -8,11 +9,16 @@ data class CourseDateBlock(
     val title: String = "",
     val description: String = "",
     val link: String = "",
+    @SerializedName("first_component_block_id")
     val blockId: String = "",
+    @SerializedName("learner_has_access")
     val learnerHasAccess: Boolean = false,
     val complete: Boolean = false,
     val date: Date?,
+    @SerializedName("date_type")
     val dateType: DateType = DateType.NONE,
+    @SerializedName("assignment_type")
+    val assignmentType: String = "",
     var dateBlockBadge: CourseDatesBadge = CourseDatesBadge.BLANK,
 ) {
```

Comment:
Could you please clarify why it's necessary to include SerializedName annotations in the domain model?
Usually, data models are used for serialization but not domain ones.

Reply:
Removed

Reply:
Thank you!